### ISSUE TITLE
Polyhedron demo: merging two point sets quit the application

### ISSUE THREAD

**Issue by cjamin** [CGAL/cgal#3067]
## Issue Details

When running the Polyhedron demo (binaries downloaded from the documentation), trying to merge 2 point sets makes the application vanish (no error message, nothing).

## Environment

* Operating system (Windows/Mac/Linux, 32/64 bits): Windows 10.
* Compiler: N/A (downloaded from CGAL doc).
* CGAL version: 4.12.

**Comment by maxGimeno** [comment]
I don't know how you even loaded two point sets, when I try it only 3 or 4 plugins load without erros.

**Comment by cjamin** [comment]
You mean, we try the binairies from the doc?
On my computer, all plugins are loaded successfully.

**Comment by maxGimeno** [comment]
@cjamin Yes, it's what I mean. I downloaded the binaries on our windows 10 x64 machine, and only a few plugins are loaded without error.

**Comment by cjamin** [comment]
@lrineau Very good question, since I made a mistake in my report: I'm using 4.11.1, not 4.12. I guess I downloaded it just a few days before the release, so I thought it was the last published version when I initiated this bug report.
With version 4.12, the merge operation works fine. And @maxGimeno: the list of loaded plugins does not show up anymore in the console, but it looks like they are successfully loaded.

**Comment by maxGimeno** [comment]
@cjamin It has been moved in the Edit->Preferences menu. Green means everything is fine, red means there are error, and IIRC gray means blcklisted.
